### PR TITLE
Add empty MB files for Lo

### LINKED ISSUE #50: Create /lo/MB/ project files
We have a partner who wants to translate MB into Lao. We need to set up the files for this in '/lo/MB/01_raw`. The files should be named and structured according to the `/th/MB/03_public` files, adjusting the yaml accordingly. 

I suggest using the Thai files and stripping out the Thai. That will leave the structure. 

```
## {MB 45.1}

###

{MB 45.1}

## {45.2}
```

(This is just an example. I haven't looked if there is a subheading under paragraph 45.1 or not.)

### PR DESCRIPTION
**Pull request by mattleff**
Fixes #50

### CODE REVIEW

**Review by bountonw — CHANGES_REQUESTED** [review]
Wrong file path. Should be in `01_raw`

File: .tooling/markdownlint/config-draft.markdownlint-cli2.cjs [hunk 1]
```diff
@@ -9,5 +9,5 @@ module.exports = {
       style: "atx",
     },
   },
-  globs: ["**/*.md", "!node_modules"],
+  globs: ["**/*.md", "!node_modules", "!lo"],
```
Comment (mattleff) [inline .tooling/markdownlint/config-draft.markdownlint-cli2.cjs:12]: @bountonw I'm excluding `lo` from linting for now since it has obvious issues.

Comment (bountonw) [reply .tooling/markdownlint/config-draft.markdownlint-cli2.cjs:12]: Shouldn't the markdown be enforced? The Thai rules won't apply, but it should check for proper markdown.

Comment (mattleff) [reply .tooling/markdownlint/config-draft.markdownlint-cli2.cjs:12]: There's lots of errors like:

```
lo/MB/01_raw/MB02_lo.md:92 MD024/no-duplicate-heading Multiple headings with the same content [Context: "###"]
lo/MB/01_raw/MB02_lo.md:146 MD003/heading-style Heading style [Expected: atx; Actual: atx_closed]
```

We can either exclude those rules or exclude the files.

Comment (bountonw) [reply .tooling/markdownlint/config-draft.markdownlint-cli2.cjs:12]: Can we exclude those specific checks that are producing the current errors?

**Review by bountonw — APPROVED** [review]

**Pull request merged**